website/src/components/navbar/navbar-client.tsx

```diff
@@ -211,7 +221,17 @@ const MobileNavigation = ({ lang, region, languages, regions, currencies, naviga
 							{translations.myProfile}
 						</NavbarLink>
 						<div className="flex-inline flex items-center">
-							{Flag && <Flag className="mx-3 h-6 w-6 rounded-full" />}
+							{country && (
+								<Image
```

Comment:
Should we also do it like this in `website/src/components/ui/currency-selector.tsx`? Then we could uninstall the package. Wdyt?

Reply:
Sure, I’ll update it.

Reply:
The URL we are using `https://raw.githubusercontent.com/lipis/flag-icons/a87d8b256743c9b0df05f20de2c76a7975119045/flags/…` only accepts country code, do you have any library suggestions for currency flags?

I would work on this on another PR since this involves with other components.

Reply:
Do you mean currency icons? Otherwise, we would have to create some mapping to map currencies to countries (similiar to the `countryToCurrency` mapping in `currency.ts`).

FYI – I created an MR (#992) to use a middleware to set the country code. Let me know what you think :)